app/lib/folio/mailer_email_templates.rb

```diff
@@ -19,15 +19,17 @@ def email_template_for!(action = nil, mailer: nil)
   end
 
   def email_template_mail(sym_data = {}, opts = {})
-    @data = sym_data.stringify_keys
+    @data = sym_data
     @site = opts.delete(:site)
     @email_template = email_template_for!(mailer: opts.delete(:mailer))
 
     @data[:ROOT_URL] = site.env_aware_root_url
     @data[:SITE_TITLE] = site.title
     @data[:DOMAIN] = site.env_aware_domain
 
-    opts[:subject] = @email_template.render_subject(@data)
+    @data = @data.stringify_keys
```

Comment:
@cabaktom proc je tady dvakrat stringify_keys, kdyz se to vsude nastavuje jako symboly?

Reply:
V metodě `email_template_mail` už stringify keys bylo, jen jsem to posunul až za přiřazení dalších hodnot do `data`, protože tak vznikaly duplicitní hodnoty když byl jednou klíč string a jednou symbol.

Devise mailer tu metodu nepoužívá, tak aby to bylo stejně, přidal jsem `stringify_keys` i do metody kterou on používá.

Vlastně ale nevím, jestli pak něco někde očekává mít ty keys jako string nebo symbol.